eq-author/src/App/section/Preview/index.js

```diff
@@ -39,6 +39,7 @@ export const UnwrappedPreviewSectionRoute = ({ match, data, loading }) => {
     <EditorLayout
       design
       preview
+      logic
```

Comment:
```suggestion
      logic = {enableOn(['hub'])}
```

Otherwise, we'll be able to see the logic page on V2 as well.

You'll need to import the enableOn function from the feature flags, too.